src/core/transaction.c

```diff
@@ -383,7 +384,7 @@ static int transaction_verify_order_one(Transaction *tr, Job *j, Job *from, unsi
 
                         /* For logging below */
                         if (strv_push_pair(&array, k->unit->id, (char*) job_type_to_string(k->type)) < 0)
-                                log_oom();
+                                (void) log_oom();
```

Comment:
Hmm, we usually don't explicitly discard log_* return values? log_oom_warning?

Reply:
Not bad, but we usually do not do like that...
And, yeah, if this is ignored, log_oom_warning() or log_oom_debug() should be better.